packages/wonder-blocks-dropdown/src/components/__tests__/search-text-input.test.js

```diff
@@ -1,145 +1,212 @@
 // @flow
 import * as React from "react";
-import {shallow} from "enzyme";
-import "jest-enzyme";
+import {render, screen} from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
 
 import SearchTextInput from "../search-text-input.js";
 
 describe("SearchTextInput", () => {
-    it("text input container should be focused when focusing on the input", () => {
+    test("text input container should be focused when focusing on the input", () => {
```

Comment:
It's great that you migrated most of these tests to RTL!

Nit: One thing to note is that some of these tests might be redundant b/c some of these cases are already tested in `search-field`, but I guess it doesn't hurt to increase testing overall!